charts/lakefs/README.md

```diff
@@ -51,6 +51,26 @@ Sensitive information like `database_connection_string` is given through "secret
 You should give your Kubernetes nodes access to all S3 buckets you intend to use lakeFS with.
 If you can't provide such access, lakeFS can be configured to use an AWS key-pair to authenticate (part of the `lakefsConfig` YAML below).
 
+## Major Chart Upgrades
+### General GuidelinesBefore 
+Before performing a major upgrade, it is strongly recommended to perform these steps:
+* Commit all uncommitted data on branches
+* Create a snapshot of your database
+
+### 0.6.1
+Introducing [lakeFS v0.80.0](https://github.com/treeverse/lakeFS/releases/tag/v0.80.0), with **Key Value Store** support. As part of this upgrade, the entire database will be ported to the new KV.
```

Comment:
What happens if I'm already upgrading to 0.6.2 (lakeFS v0.81.0)?

Reply:
We will have to solve this when we get there.
It will either be a 2 step upgradte that must go through 0.6.0, same with upgrading lakeFS 0.70.0 to lakeFS 0.81.0, or if we can somhow leverage helm to do it for us implicitly (e.g. running 2 upgrade jobs, one with lakeFS 0.80.0, hard-coded, to get the migration done, and then a second job to get to the target lakeFS)
I don't have a really good solution at the moment, but I don;t think it is a blocker for the current upgrade